extension/js/common/core/common.ts

```diff
@@ -283,7 +283,12 @@ export class Str {
   };
   private static formatEmailWithOptionalNameEx = ({ email, name }: EmailParts, forceBrackets?: boolean): string => {
     if (name) {
-      return `${Str.rmSpecialCharsKeepUtf(name, 'ALLOW-SOME')} <${email}>`;
+      /*
+       * hotfix for issue https://github.com/FlowCrypt/enterprise-server/issues/6264
+       * this replaces accented characters with their unaccented counterparts
+       */
+      const normalizedName = name.normalize('NFD').replace(/[\u0300-\u036f]/g, '');
+      return `${Str.rmSpecialCharsKeepUtf(normalizedName, 'ALLOW-SOME')} <${email}>`;
```

Comment:
`formatEmailWithOptionalNameEx` used for common encrypted emails too.

to replace accented characters only for password-protected messages let's create a separate method `Str.replaceAccentedChars` and use it in `webPortalMessageUpload` as `to: (recipients.to || []).map(Str.formatEmailWithOptionalName).map(Xss.stripEmojis).map(Str.replaceAccentedChars)`

Reply:
Thanks, Roma, I notice that it is indeed incorrect. I'll be fixing it.